src/vivarium_gates_mncnh/constants/data_keys.py

```diff
@@ -79,7 +79,11 @@ def log_name(self):
 
 class __ANC(NamedTuple):
     # Keys that will be loaded into the artifact. must have a colon type declaration
-    ESTIMATE: str = "covariate.antenatal_care_1_visit_coverage_proportion.estimate"
+    ANCfirst: str = (
```

Comment:
Nit: ANCFirst or ANC_First

Reply:
I think ANC_first would look better too but I'm prioritizing staying very consistent with the [documentation names](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.html#id3)